db/stop_database.go

```diff
@@ -32,15 +31,30 @@ const (
 // Shutdown :: closes the client connection and stops the
 // database instance if the given `invoker` matches
 func Shutdown(client *Client, invoker Invoker) {
-	log.Println("[TRACE] shutdown")
+	utils.LogTime("db.Shutdown start")
+	defer utils.LogTime("db.Shutdown end")
+
 	if client != nil {
 		client.Close()
 	}
 
 	status, _ := GetStatus()
 
-	// force stop if the service was invoked by the same invoker and we are the last one
-	if status != nil && status.Invoker == invoker {
+	if status != nil {
+		if status.Invoker == InvokerService {
```

Comment:
comment explaining logic here - why nothing to do?

Reply:
explained